analyses/helpers/RunningStats.py

```diff
@@ -0,0 +1,37 @@
+## Class for calculating a running SD and mean.
+import math
+
+class RunningStats:
```

Comment:
Is the purpose of this class to save memory by just updating the numbers as the randomizations are processed?  If so, could you add a line or two of documentation explaining that.

I have written a couple of tools for managing the computations from sets of randomizations and I will likely merge this into that code once it is available to this repository.